### PR TITLE
Enhances Social Sharing Module by sending GA social interaction data

### PR DESCRIPTION
**Pull request by inetbiz**
This begins work on issue #1  to add GA social interactions .onclick.

### CODE REVIEW

**Comment by inetbiz** [comment]
@firstred would you review the JS for me? I'm still testing GA results on a client site with these changes.

File: js/socialsharing.js [hunk 1]
```diff
@@ -32,19 +32,25 @@ $(document).ready(function(){
 			{
 				case 'twitter':
 					window.open('https://twitter.com/intent/tweet?text=' + sharing_name + ' ' + encodeURIComponent(sharing_url), 'sharertwt', 'toolbar=0,status=0,width=640,height=445');
+                                        ga('send', 'social', 'Twitter', 'share', ' + sharing_url + ');
```
Comment (inetbiz) [inline js/socialsharing.js]: `sharing_url` did not populate when sent to `ga()`

**Comment by inetbiz** [comment]
I updated it to send the name of the entity, `sharing_name` because you can use secondary dimensions in your GA report to show the URL shared. This PR is ready for merging, @firstred

**Comment by inetbiz** [comment]
https://www.screencast.com/t/aIKvHPv77Qm

**Comment by inetbiz** [comment]
https://content.screencast.com/users/Denver_Prophit_Jr/folders/Snagit/media/9187af95-5af5-411e-940d-76d5f0168788/04.21.2017-13.20.png

File: js/socialsharing.js [hunk 2]
```diff
@@ -32,19 +32,25 @@ $(document).ready(function(){
...
 					window.open('https://plus.google.com/share?url=' + sharing_url, 'sharer', 'toolbar=0,status=0,width=660,height=445');
+					// GA doesn't need to be added as GA will automaticaly track G+ social interactions
 					break;
 				case 'pinterest':
 					var img_url = sharing_img;
 					if (typeof $("#bigpic").attr('src') != 'undefined' && $("#bigpic").attr('src') != '') {
 						img_url = $("#bigpic").attr('src');
 					}
 					window.open('http://www.pinterest.com/pin/create/button/?media=' + img_url + '&url=' + sharing_url, 'sharerpinterest', 'toolbar=0,status=0,width=660,height=445');
+                                        ga('send', 'social', 'Pinterest', 'share', sharingname);
```
Comment (eschiendorfer) [inline js/socialsharing.js]: Shouldn't it be "sharing_name"?

**Comment by firstred** [comment]
> This PR is ready for merging

It is! 
What a great contribution, Denver! Some real good work.

**Pull request merged**